### ISSUE TITLE
Needed to modify default Dockerfile to install laminas MVC skeleton

### ISSUE THREAD

**Issue by sergarbes** [laminas/laminas-mvc-skeleton#14]
### Bug Report

|    Q        |   A
|------------ | ------
| Version(s)  | x.y.z

#### Summary

I am trying the installation of the MVC skeleton withc docker, vagrant and with apache. I have started with docker and to succes I had to modify the Dockerfile with the Intl PHP extension and the libicu-dev package. As forms support depends on i18n support it seams that Intl is a PHP requirement for the docker.

#### Current behavior

The installation crashes.

When the installation process install the i18n support it crashes with a PHP error that  points me to the installation of Intl package and libicu-dev.

#### How to reproduce

I am using a MacOS Catalina with the Catalan language setted up.

$ composer self-update
$ composer install

With the original Dockerfile.

FROM php:7.3-apache

*RUN apt-get update *

&& apt-get install -y git zlib1g-dev libzip-dev *
&& docker-php-ext-install zip *
&& a2enmod rewrite *
&& sed -i ‘s!/var/www/html!/var/www/public!g’ /etc/apache2/sites-available/000-default.conf *
&& mv /var/www/html /var/www/public *
&& curl -sS https://getcomposer.org/installer *
| php – --install-dir=/usr/local/bin --filename=composer*

WORKDIR /var/www

The installation crashes.

#### Expected behavior

My actual Dockerfile is:

FROM php:7.3-apache

*RUN apt-get update *

&& apt-get install -y git zlib1g-dev libzip-dev libicu-dev *
&& docker-php-ext-install zip *
&& docker-php-ext-configure intl *
&& docker-php-ext-install intl *
&& a2enmod rewrite *
&& sed -i ‘s!/var/www/html!/var/www/public!g’ /etc/apache2/sites-available/000-default.conf *
&& mv /var/www/html /var/www/public *
&& curl -sS https://getcomposer.org/installer *
| php – --install-dir=/usr/local/bin --filename=composer*

WORKDIR /var/www

A successful installation with the prompt showing:

Do you want a minimal install (no optional packages)? Y/n

**Comment by anukaal** [comment]
Try to work on the virtual box and on other OS , I think it should work

**Comment by samsonasik** [comment]
should already fixed by #20 . lib-icu can be enable by uncomment it in Dockerfile https://github.com/laminas/laminas-mvc-skeleton/blob/df7ec944ce1a041b155ed64ddbf31431225e2a04/Dockerfile#L32

Please create new issue if problem persist.

**Comment by sergarbes** [comment]
Ok, Thanks

Sergi

**Comment by sergarbes** [comment]
Ok, Thanks!

Sergi
